data/zaidahmad25.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Zaid Ahmad",
+  "type": "personal",
+  "bio": "Empowering growth and learning through my passion for web development and open source ❤️",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/zaidahmad25",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "My GitHub",
+      "url": "https://github.com/zaidahmad25",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/zaidahmad25",
+      "icon": "FaLinkedinIn"
```

Comment:
Change the icon to `SiLinkedin`
``` suggestion
      "icon": "SiLinkedin"
```

Reply:
Hi there! I don't know how to change the contents of the file once a user makes a pull request. So please guide me for the same.

Reply:
@priyankarpal the icon `FaLinkedinIn` is a valid icon (see screenshot) so this should not be a blocker for merging the PR. 

<img width="341" alt="Screenshot 2023-02-12 at 09 48 16" src="https://user-images.githubusercontent.com/82668196/218290223-8a3e5d95-6141-4629-8090-c622cca2a756.png">



> Hi there! I don't know how to change the contents of the file once a user makes a pull request. So please guide me for the same.

@zaidahmad25 you do not need to change the contents of your file if you want to keep this icon.  For future reference, here is a link to our Docs which guides you how to make changes (see the Open Pull Request section in answer to your question): https://linkfree.eddiehub.io/docs/how-to-guides/editing

Reply:
Thank you!